lightning/src/ln/channel.rs

```diff
@@ -1040,7 +1053,7 @@ impl<Signer: Sign> Channel<Signer> {
 	/// Note that below-dust HTLCs are included in the fourth return value, but not the third, and
 	/// sources are provided only for outbound HTLCs in the fourth return value.
 	#[inline]
-	fn build_commitment_transaction<L: Deref>(&self, commitment_number: u64, keys: &TxCreationKeys, local: bool, generated_by_local: bool, logger: &L) -> (CommitmentTransaction, u32, usize, Vec<(HTLCOutputInCommitment, Option<&HTLCSource>)>) where L::Target: Logger {
+	fn build_commitment_transaction<L: Deref>(&self, commitment_number: u64, keys: &TxCreationKeys, local: bool, generated_by_local: bool, logger: &L) -> (CommitmentTransaction, u32, usize, Vec<(HTLCOutputInCommitment, Option<&HTLCSource>)>, u64, u64) where L::Target: Logger {
```

Comment:
Fine to do this in follow-up, I think it'd be less error-prone to `struct`-ify this return value

Reply:
See newly introduced `CommitmentStats` :)